fir.lic

```diff
@@ -0,0 +1,171 @@
+custom_require.call(%w[common common-crafting common-items common-money common-travel drinfomon equipmanager])
```

Comment:
Would you mind adding a documentation block as a header?

Reply:
done!